tests/algorithms/test_mce_irl.py

```diff
@@ -240,36 +240,37 @@ def test_tabular_policy():
     pi = np.stack(
         [np.eye(2), 1 - np.eye(2)],
     )
-    rng = np.random.RandomState(42)
+    random_state = np.random.RandomState(42)
```

Comment:
Use fixture instead?

I don't think the test needs the seed to be 42. I hope not!